### PR TITLE
updates suricata to assign SCC to SA and update image

### PR DESCRIPTION
**Pull request by tonytheleg**
### What type of PR is this?
_(bug/feature/cleanup/documentation)_

Feature

### What this PR does / why we need it?

* Updaets the FR suricata daemonset to use new SCC changes and ensure its assigned to SA
* Bumps image to ensure latest code changes and CVE patches

### Which Jira/Github issue(s) this PR fixes?

For [OSD-17946](https://issues.redhat.com//browse/OSD-17946)

### Pre-checks (if applicable):
- [x] Tested latest changes against a cluster
- [x] Included documentation changes with PR
- [x] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

File: deploy/osd-suricata/05-osd-suricata-scc-SecurityContextConstraint.yaml [hunk 1]
```diff
@@ -21,6 +21,8 @@ seLinuxContext:
   type: RunAsAny
 supplementalGroups:
   type: RunAsAny
+users:
+- suricata-sa  
 volumes:
 - emptyDir
```
Comment (dustman9000) [inline deploy/osd-suricata/05-osd-suricata-scc-SecurityContextConstraint.yaml:27]: You'll want all volumes as well.

File: deploy/osd-suricata/05-osd-suricata-scc-SecurityContextConstraint.yaml [hunk 2]
```diff
@@ -21,6 +21,8 @@ seLinuxContext:
   type: RunAsAny
 supplementalGroups:
   type: RunAsAny
+users:
+- suricata-sa
```
Comment (dustman9000) [inline deploy/osd-suricata/05-osd-suricata-scc-SecurityContextConstraint.yaml:25]: You need to rename the SA manifest to `04-...` so the SA is ensured to be created before applying this.

File: deploy/osd-suricata/20-osd-suricata-Daemonset.yaml [hunk 3]
```diff
@@ -33,7 +33,7 @@ spec:
         - mountPath: /host/
```
Comment (dustman9000) [inline deploy/osd-suricata/20-osd-suricata-Daemonset.yaml:33]: Change to `/host/var/log` and fix the volume to point to `/var/log`

File: deploy/osd-suricata/05-osd-suricata-scc-SecurityContextConstraint.yaml (hunk 1 above)
Comment (tonytheleg) [reply deploy/osd-suricata/05-osd-suricata-scc-SecurityContextConstraint.yaml]: Updated

File: deploy/osd-suricata/05-osd-suricata-scc-SecurityContextConstraint.yaml (hunk 2 above)
Comment (tonytheleg) [reply deploy/osd-suricata/05-osd-suricata-scc-SecurityContextConstraint.yaml]: Updated

File: deploy/osd-suricata/20-osd-suricata-Daemonset.yaml (hunk 3 above)
Comment (tonytheleg) [reply deploy/osd-suricata/20-osd-suricata-Daemonset.yaml]: Updated

**Pull request merged**